components/browser/engine-system/src/main/java/mozilla/components/engine/gecko/SystemEngine.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.engine.gecko
```

Comment:
This should be engine.system? Sorry, might have missed that before.

Reply:
Argh. I think what happened is that Android Studio didn't only rename the package in the module I'm currently in but also renames it in all other modules. That's unexpected.